plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/MongoQueryRunner.java

```diff
@@ -81,7 +83,12 @@ public static Session createSession()
 
     public static MongoClient createMongoClient(MongoServer server)
     {
-        return new MongoClient(server.getAddress().getHost(), server.getAddress().getPort());
+        MongoClientSettings.Builder options = MongoClientSettings.builder();
+
+        options.applyToClusterSettings(builder ->
+                builder.hosts(ImmutableList.of((new ServerAddress(server.getAddress().getHost(), server.getAddress().getPort())))));
+
+        return MongoClients.create(options.build());
```

Comment:
Please add `ConnectionString getConnectionString()` to `MongoServer` and use it.